qsirecon/interfaces/dipy.py

```diff
@@ -183,6 +183,8 @@ def _extrapolate_scheme(self, scheme_name, runtime, fit_obj, mask_img):
 
         gtab_to_predict = self._get_gtab(external_bvals=bval_file, external_bvecs=bvec_file)
         new_data = fit_obj.predict(gtab_to_predict, S0=1)
```

Comment:
Unrelated, but since we're here could we add s0=1000?

Reply:
Done!